factory/ideas.php

```diff
@@ -730,4 +746,23 @@ public function get_idea_count()
 	{
 		return $this->idea_count;
 	}
+
+	/**
+	 * Helper to generate the user profile URL with an
+	 * absolute URL, which helps avoid problems when
+	 * used in AJAX requests.
+	 *
+	 * @return string User profile URL
+	 */
+	protected function profile_url_helper()
+	{
+		static $url;
```

Comment:
I don't know about this. I would add a property for this. Thought static variables in functions are not cool anymore.